src/desktop.c

```diff
@@ -268,6 +268,43 @@ desktop_focus_topmost_mapped_view(struct server *server)
 	desktop_move_to_front(view);
 }
 
+void
+desktop_focus_output(struct output *output)
+{
+	if (!output || output->server->input_mode != LAB_INPUT_STATE_PASSTHROUGH) {
```

Comment:
Maybe replace `!output` with `!output_is_usable(output)` to prevent trying to warp the cursor to a disabled output?

Reply:
Hmm, I missed the `output_is_usable()` check in `output_from_name()`. Still it might not hurt to check again here, to make `desktop_focus_output()` more independent.

Reply:
Added the additional check.